src/lib/widget/_utilities/api.utilities.ts

```diff
@@ -139,8 +144,8 @@ export function widgetApiFactory(componentApi: ReturnType<typeof _componentApi>)
       logErrorAndThrow('missingStores');
     }
 
-    const requestsOauth = options?.oauth || true;
```

Comment:
Was this causing an issue? I'm  curious i would have expected this to still work just fine?

Reply:
If the customer does not want the widget to request `user` or `oauth`, they would pass `false` to these config properties, which leads to the right-hand operand being assigned. This means the user essentially can never disable these features, so that's why I switched to the nullish coalescing operator.

Reply:
ah, sneaky, sneaky. makes sense.